action/helper_test.go

```diff
@@ -6,11 +6,23 @@ import (
 	"path/filepath"
 	"reflect"
 	"testing"
+
+	"github.com/deis/helm/log"
 )
 
+const tmpConfigfile = `repos:
+  default: charts
+  tables:
+    - name: charts
+      repo: https://github.com/deis/charts
+`
+
 var helmRoot string
 
 func init() {
+	// Turn on debug output, convert os.Exit(1) to panic()
+	log.IsDebugging = true
```

Comment:
Actually, this is done for an interesting reason: if IsDebugging is true, instead of calling `os.Exit(1)`, `log.Die` calls `panic(err)` and gives a nice pretty stack trace